components/jaeger-collector.libsonnet

```diff
@@ -7,6 +7,10 @@ local service = k.core.v1.service;
     namespace:: error 'must set namespace for jaeger',
     image:: error 'must set image for jaeger',
     replicas:: 1,
+    pvc:: {
+      class: 'standard',
```

Comment:
Do we know if `standard` is some kinda standard (sorry) storageClass that people use in K8s?

Reply:
Might be, yes. However it's better than nothing, I believe. So we don't really have a choice, but can tell people to change it similar to the way we set the storage class in openshift.

Reply:
Sounds good :+1: